src/main/tv/codely/scala_intro_examples/lesson_10_enums_vs_sealed_structures/exercise_solutions/daniel_albarral/UsageExample.sc

```diff
@@ -0,0 +1,41 @@
+/**
+  * This doesn't work with IntelIJ WS for reasons that
+  * I don't understand.
+  * --> Run it in a repel ! <--
+  * If some one have some idea of what's happening
+  * any help is welcome.
```

Comment:
Yes. Probably it doesn't work because you're not compiling the whole project while running the Worksheet.

![scala_intro_example 2018-03-12 13-34-45](https://user-images.githubusercontent.com/986235/37283971-2a4e35ca-25fa-11e8-9c5e-442de40cd96b.png)

If you don't check this checkbox, the IDE will not compile other classes such as the `Food` one, and therefore the Worksheet wouldn't be able to be executed.